cs-amazon/src/main/java/io/cloudslang/content/amazon/actions/cloudformation/CreateStackAction.java

```diff
@@ -1,115 +1,131 @@
+     * @param proxyPort         Optional - proxy server port. You must either specify values for both proxyHost and
+     *                          proxyPort inputs or leave them both empty.
+     *                          Default: ""
+     * @param proxyUsername     Optional - proxy server user name.
+     *                          Default: ""
+     * @param proxyPassword     Optional - proxy server password associated with the proxyUsername input value.
+     *                          Default: ""
+     * @param region            AWS region name
+     *                          Example: "eu-central-1"
+     * @param stackName         Stack name to create
+     *                          Example: "MyStack"
+     * @param templateBody      AWS Cloud Formation template body in JSON or YAML format
+     * @param parameters        AWS Cloud Formation template parameters in key value format, one key=value per line
+     * @return                  A map with strings as keys and strings as values that contains: outcome of the action, returnCode of the
+     *                          operation, or failure message and the exception if there is one
+     */
+    @Action(name = "Create AWS Cloud Formation Stack",
+            outputs = {
+                    @Output(Outputs.RETURN_CODE),
+                    @Output(Outputs.RETURN_RESULT),
+                    @Output(Outputs.EXCEPTION)
+            },
+            responses = {
+                    @Response(text = Outputs.SUCCESS, field = Outputs.RETURN_CODE, value = Outputs.SUCCESS_RETURN_CODE,
+                            matchType = MatchType.COMPARE_EQUAL, responseType = ResponseType.RESOLVED),
+                    @Response(text = Outputs.FAILURE, field = Outputs.RETURN_CODE, value = Outputs.FAILURE_RETURN_CODE,
+                            matchType = MatchType.COMPARE_EQUAL, responseType = ResponseType.ERROR)
+            }
+    )
+    public Map<String, String> execute(
+            @Param(value = IDENTITY,   required = true)  String identity,
+            @Param(value = CREDENTIAL, required = true, encrypted = true)  String credential,
+            @Param(value = REGION,     required = true)  String region,
+            @Param(value = PROXY_HOST)                   String proxyHost,
+            @Param(value = PROXY_PORT)                   String proxyPort,
+            @Param(value = PROXY_USERNAME)               String proxyUsername,
+            @Param(value = PROXY_PASSWORD, encrypted = true) String proxyPassword,
+            @Param(value = CONNECT_TIMEOUT)              String connectTimeoutMs,
+            @Param(value = EXECUTION_TIMEOUT)            String execTimeoutMs,
+            @Param(value = STACK_NAME, required = true)  String stackName,
+            @Param(value = TEMPLATE_BODY, required = true) String templateBody,
+            @Param(value = PARAMETERS)                   String parameters) {
+
+        proxyPort = defaultIfEmpty(proxyPort, DefaultValues.PROXY_PORT);

```

Comment:
I would personally use here different names for constants in **DefaultValues** so they will be different than the ones in **CommonInputs**. This way you can static import **DefaultValues**. For example, I would use something like **PROXY_PORT_VALUE**.